src/cmd/node/mod.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Result;
+
+mod rpc;
+
+/// Commands for accessing the node.
+#[derive(clap::Subcommand)]
+pub enum Node {
+    #[clap(subcommand, about = "Access node's RPC requests")]
+    Rpc(Rpc),
+}
+
+impl Node {
+    /// Executes the command.
+    pub fn run(&mut self) -> Result<()> {
```

Comment:
Maybe that should be `execute` instead of `run` like for the subcommands. Then we could be consistent and name it everywhere the same.
Even the comment says it executes something :smile: